### PR TITLE
qase-api-client: add a new client

### PR DESCRIPTION
**Pull request by gibiw**
qase-api-client: add a new client
--
Add Qase API v1 client

---

ci: add a pipeline
--
Add the pipeline for go packages

### CODE REVIEW

File: .github/workflows/gopackage.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          list-files: 'shell'
+          filters: |
+            changes:
+              - '${{ matrix.changed-dir }}/**'
+      - name: Set up Go ${{ matrix.go-version }}
+        if: steps.filter.outputs.changes == 'true'
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Install dependencies
```
Comment (NickVolynkin) [inline .github/workflows/gopackage.yml:29]: let's keep blank lines between steps
```suggestion
          go-version: ${{ matrix.go-version }}
          
      - name: Install dependencies
```

File: .github/workflows/gopackage.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Python package
```
[the remark this answers is not in the document]
Comment (NickVolynkin) [reply .github/workflows/gopackage.yml:1]: or even Go build, because it doesn't do packaging

**Review by NickVolynkin — APPROVED** [review]

File: .github/workflows/gopackage.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gibiw) [reply .github/workflows/gopackage.yml]: fixed

File: .github/workflows/gopackage.yml (hunk 1 above)
Comment (gibiw) [reply .github/workflows/gopackage.yml]: fixed

**Pull request merged**